chiabip158/__init__.py

```diff
@@ -0,0 +1,2 @@
+
+from ._chiabip158 import PyBIP158
```

Comment:
related to the comments about the `.pyi` naming, this may need to be indicated for explicit re-export.  maybe only in 3.13+?  i'm not sure.

there are options, but this is a simple one to consider in this small case.  yes, the repetition is annoying.


```suggestion
from ._chiabip158 import PyBIP158 as PyBIP158
```